ocaml/testsuite/tests/templates/basic/test.ml

```diff
@@ -13,6 +13,8 @@ module = "monoid.mli"
 module = "bad_ref_direct.ml"
 compiler_output = "bad_ref_direct.output"
 ocamlc_byte_exit_status = "2"
-**** check-ocamlc.byte-output
+**** skip
+reason = "error currently broken"
```

Comment:
Oh - right, this deserved better explanation. Basically, the error we want to give is “this should be declared as a parameter,” but since the `-parameter` parameter isn't implemented yet, the error is just a fatal error with a traceback since it's an error for _anyone_ to reference a parameter module _at all_. (It would be better to say the error _message_ is broken, I suppose. The test still checks that we exit with status 2, all that's being skipped is the diff on the error message.)

Reply:
Improved the `reason` field.